src/Event/Decorator/SubjectFilterDecorator.php

```diff
@@ -34,11 +34,11 @@ class SubjectFilterDecorator extends AbstractDecorator
     /**
      * @inheritDoc
      */
-    public function __invoke()
+    public function __invoke(): mixed
     {
         $args = func_get_args();
         if (!$this->canTrigger($args[0])) {
-            return false;
+            return null;
```

Comment:
Is this supposed to be `false`? if so, should `ConditionalDecorator` return null or false? `false` means to stop propagation of the event while null means don't set the result, right?

Reply:
Yes, the `false` does seem odd. Returning `null` makes more sense.